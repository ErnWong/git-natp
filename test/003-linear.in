----A----B--C--LongCommit01234Name--+++--------<<<>>>-----D.E,_'`^*-~-----F master
