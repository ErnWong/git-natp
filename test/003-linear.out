A
B A
C B
LongCommit01234Name C
D LongCommit01234Name
E D
F E
